### PR TITLE
Card Generation Contract and Tests

### PR DESCRIPTION
**Pull request by daywhirls**
- Added GenerateCard.sol to contracts
- Added Migration file for GenerateCard
- Updated all createCard parameters for _attributes value
- Added testcases script for GenerateCard

### CODE REVIEW

**Comment by nickysemenza** [comment]
grr, ignore circleci @daywhirls , or actually, updating your branch from master should make it go away

since this is a lot of commits for just one feature, might be good to squash these when you merge 

image

File: eth_contracts/contracts/GenerateCard.sol [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+contract GenerateCard {
+
+    CardOwnership public cardOwnership = new CardOwnership();
+
+    function initialGeneration(uint256 numberOfCards) external returns (uint256 isSuccess) {
+        uint256 i;
+        for (i = uint256(1); i <= numberOfCards; i++) {
+            uint256 returnedID = 1;
+            cardOwnership.createCard(this, i);
+            //cardOwnership.totalSupply();
```
Comment (harrischristiansen) [inline eth_contracts/contracts/GenerateCard.sol]: Delete this line

**Review by harrischristiansen — CHANGES_REQUESTED** [review]

**Comment by nickysemenza** [comment]
oh and `backend/composer.phar` shouldn't be here...

File: eth_contracts/contracts/GenerateCard.sol (hunk 1 above)
Comment (daywhirls) [reply eth_contracts/contracts/GenerateCard.sol]: Yes my lord

**Review by harrischristiansen — APPROVED** [review]
https://github.com/CryptoCardsETH/cryptocards/issues/4

**Pull request merged**